chapter_02_repository.asciidoc

```diff
@@ -627,12 +627,14 @@ def test_repository_can_retrieve_a_batch_with_allocations(session):
 //TODO (DS): Picking a descriptive SKU (e.g. 'comfy-sofa') would make this a
 //bit more fun to read.
 // Worth explaining why we have to do a follow up query to get the id inserted?j
-// Why the underscore in _allocations here?
+// Why the underscore in _allocations here? It was already private in the definition
+// of the Batch class in chapter 1. Maybe for consistency we want to make them all
+// private ('_') and explain that we want to access them through properties for better control?
 
 Whether or not you painstakingly write tests for every model is a judgement
 call.  Once you have one class tested for create/modify/save, you might be
 happy to go on and do the others with a minimal roundtrip test, or even nothing
-at all, if they all follow a similar pattern.  In our case, the ORM config
+at all, if they all follow a similar pattern. In our case, the ORM config
```

Comment:
ok look, if I want to use two spaces after a full stop will you let me please? I mean I really don't want to dissuade you from writing these comments which I really, really appreciate, but these spaces don't even render in the published text! two spaces after a full stop is in my muscle memory...

Reply:
Okay, I will try to focus more on the important bits 🤘